backend/package.json

```diff
@@ -1,9 +1,21 @@
 {
+  "devDependencies": {
+    "standard": "*"
+  },
+  "scripts": {
+    "test": "standard"
+    
```

Comment:
Awesome, awesome work!! Quick little change tho: add a start script in this file with the command to start up the server. 

Reasoning - 
- provides a more centralized way of starting up the server so you only gotta run npm start when someone cd's into the folder.
- Ensures that in the case that these files change or the main server file is no longer app.js in the future, it will be a one liner change in the package file.

Code right above the test script:
"start": "nodemon app.js"